tests/gdk/commands/test/config/test_InitConfiguration.py

```diff
@@ -68,6 +68,42 @@ def test_GIVEN_gdk_config_with_gtf_and_otf_version_WHEN_test_init_THEN_use_gtf_v
         init_config = InitConfiguration({})
         assert init_config.otf_version == "1.2.3"
 
+    def test_GIVEN_gdk_config_with_gtf_version_WHEN_test_init_with_otf_version_arg_THEN_use_arg_version(self):
+        config = self._get_config(
+            {
+                "test-e2e": {
+                    "gtf_version": "1.1.0",
+                }
+            }
+        )
+        self.mocker.patch("gdk.common.configuration.get_configuration", return_value=config)
+        init_config = InitConfiguration({"otf_version": "1.2.3"})
+        assert init_config.otf_version == "1.2.3"
+
+    def test_GIVEN_gdk_config_with_gtf_version_WHEN_test_init_with_gtf_version_arg_THEN_use_arg_version(self):
+        config = self._get_config(
+            {
+                "test-e2e": {
+                    "gtf_version": "1.1.0",
+                }
+            }
+        )
+        self.mocker.patch("gdk.common.configuration.get_configuration", return_value=config)
+        init_config = InitConfiguration({"gtf_version": "1.2.3"})
+        assert init_config.otf_version == "1.2.3"
```

Comment:
nit: this parameter is still called `otf_version`

Reply:
Yeah, some internal functions still reference it as otf, will change all references in a later PR.